core/src/main/resources/schema/app/20200824205107_CB-1068_multi_az.sql

```diff
@@ -0,0 +1,49 @@
+-- // CB-1068 multi az
+-- Migration SQL that makes the change goes here.
+
+CREATE TABLE IF NOT EXISTS instancegroupnetwork (
+    id bigint NOT NULL,
+    cloudPlatform CHARACTER VARYING(255),
+    attributes TEXT,
+    stack_network_id bigint,
+    instancegroup_id bigint,
+    subnetid TEXT,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE instancegroup_availabilityzones (
+    instancegroup_id bigint NOT NULL,
+    availabilityzone text,
+    PRIMARY KEY (instancegroup_id)
+);
+
+ALTER TABLE instancegroup ADD COLUMN instancegroupnetwork_id bigint;
+ALTER TABLE ONLY instancegroupnetwork ADD CONSTRAINT fk_instancegroup_network_id FOREIGN KEY (instancegroup_id) REFERENCES instancegroup(id);
+CREATE SEQUENCE IF NOT EXISTS instancegroupnetwork_id_seq START WITH 1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+INSERT INTO instancegroup_availabilityzones (instancegroup_id, availabilityzone) SELECT instancegroup.id, availabilityzone FROM instancegroup INNER JOIN stack ON stack.id = instancegroup.stack_id WHERE stack.availabilityzone IS NOT NULL;
+INSERT INTO instancegroupnetwork (id, cloudPlatform, instancegroup_id, stack_network_id) SELECT nextval('instancegroupnetwork_id_seq') AS "id", cloudplatform, instancegroup.id, stack.network_id FROM instancegroup INNER JOIN stack ON stack.id = instancegroup.stack_id;
+UPDATE instancegroupnetwork SET attributes=network.attributes FROM network WHERE network.id=instancegroupnetwork.stack_network_id;
+
+-- Search subnet-id
+UPDATE instancegroupnetwork SET subnetid=instancegroupnetwork.attributes::json->'subnetId';
+UPDATE instancegroupnetwork SET subnetid = REPLACE(subnetid,'"','');
+
+-- If AWS, AZURE, GCP, OPENSTACK, MOCK
+UPDATE instancegroupnetwork SET attributes = json_build_object('subnetIds', json_build_array(subnetid), 'cloudPlatform', cloudPlatform) WHERE subnetid IS NOT NULL;
+
+-- If YCLOUD
+UPDATE instancegroupnetwork SET attributes = json_build_object('subnetIds', json_build_array(), 'cloudPlatform', cloudPlatform) WHERE subnetid IS NULL;
+UPDATE instancegroup ig SET instancegroupnetwork_id=ign.instancegroup_id FROM instancegroupnetwork ign WHERE ign.instancegroup_id = ig.id;
+
+-- Cleaning up tmp columns
+ALTER TABLE instancegroupnetwork DROP COLUMN stack_network_id;
+ALTER TABLE instancegroupnetwork DROP COLUMN subnetid;
+ALTER TABLE instancegroupnetwork DROP COLUMN IF EXISTS instancegroup_id;
```

Comment:
How about dropping `CONSTRAINT fk_instancegroup_network_id` on `instancegroupnetwork` explicitly before removing the column `instancegroup_id`?